rj_gameplay/stp/skill/rj_sequence.py

```diff
@@ -0,0 +1,33 @@
+import py_trees
+import random
+import stp.action as action
+from typing import Optional, Any
+import stp.rc as rc
+from typing import List
+from stp.skill.action_behavior import ActionBehavior, RobotActions
+
+class RjSequence(py_trees.composites.Sequence):
+    """
+    A sequence block for our use
+    This is currently needed since we need a way to send actions back up the tree
```

Comment:
Sorry, I'm a little confused as to what "sending actions back up the tree" means here. It looks like the sequence just ticks through each of its actions one-by-one as they finish; what part requires the sending back up?

Reply:
Ah actually that comment is wrong, its actually for sending world state and robot down the tree since we need the world state to use the is_done() command.

Reply:
Gotcha